VictoriaAlbert/VictoriaAlbert/VictoriaAlbertTableViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  VictoriaAlbertTableViewController.swift
+//  VictoriaAlbert
+//
+//  Created by Marcel Chaucer on 11/10/16.
+//  Copyright © 2016 Marcel Chaucer. All rights reserved.
+//
+
+import UIKit
+
+class VictoriaAlbertTableViewController: UITableViewController, UISearchBarDelegate {
+    var objects: [MuseumObject] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        createSearchBar()
+        APIRequestManager.manager.getData(endPoint: "http://www.vam.ac.uk/api/json/museumobject/search?q=swords") { (data: Data?) in
+            if  let validData = data,
+                let validInfo = MuseumObject.objects(from: validData)
+                 {
+                self.objects = validInfo
+                DispatchQueue.main.async {
+                    self.tableView.reloadData()
+                }
+            }
+        }
+    }
+    
+    func createSearchBar() {
+        let searchbar = UISearchBar()
+        searchbar.showsCancelButton = false
+        searchbar.placeholder = "Enter object to search"
+        searchbar.delegate = self
+        self.navigationItem.titleView = searchbar
+    }
+    
+    func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
+        guard let searchTerm = searchBar.text else {return}
+        APIRequestManager.manager.getData(endPoint: "http://www.vam.ac.uk/api/json/museumobject/search?q=\(searchTerm)") { (data: Data?) in
```

Comment:
Search bar is great. You should refactor these two calls (the default sword one and this search one) into one method you call from both. (Nice choice of search term too. I notice the Tolkien and (I think) Game of Thrones sympathies. Never hurts to be a nerd (about something -- anything) in this biz.)